### DISCUSSION TITLE
AD Network

### DISCUSSION THREAD

**Discussion by IbrahimSemab** [godotengine/godot-proposals#5743] [Engine Core]
I wondered that godot had its own ad network because i am having issue with google admob it is showing payment error because i did not added any tax information and I literally had no idea how to add tax information and due to that my account got blocked. It is really hard to manage. I am a godot beginner but I know the basics of godot very well. Any Solution?

**Comment by ghost** [comment]
> I wondered that godot had its own ad network because i am having issue with google admob it is showing payment error because i did not added any tax information and I literally had no idea how to add tax information and due to that my account got blocked. It is really hard to manage. I am a godot beginner but I know the basics of godot very well. Any Solution?

You can search on YouTube or Google. Like
Admob tax information.

I got 2 Solutions for india and Bangladesh..

**Comment by awardell** [comment]
Are you using this? https://github.com/Shin-NiL/Godot-Android-Admob-Plugin

Perhaps you should bring this discussion over there.
